playbooks/slurm-cluster/slurm-validation.yml

```diff
@@ -1,33 +1,212 @@
 ---
-# Playbook designed to run a NCCL test across all nodes in a cluster of DGX-1s
+# Playbook designed to run a NCCL test across nodes in a cluster of DGXs
+# Example to run on NSL-B with two nodes:
```

Comment:
We shouldn't have references to internal clusters if it can be avoided.

Reply:
I can reword to:
```
# Example to run on two nodes with enp set for out-of-band NCCL init, no UCX and no HCOLL:
```

Or please suggest how to reword. This example will work in clusters without RDMA (no RoCE or infiniband). I want to document in the comments how to run on such clusters. Sometimes without the RDMA setup it is useful to test such cases.

Reply:
Fixed in commit af08cba.